<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicervolume_mathModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicervolume_mathModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>volume_math</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="1">
           <widget class="qMRMLNodeComboBox" name="inputAVolumeNodeSelector">
            <property name="nodeTypes">
             <stringlist notr="true"/>
            </property>
            <property name="hideChildNodeTypes">
             <stringlist notr="true"/>
            </property>
            <property name="addEnabled">
             <bool>false</bool>
            </property>
            <property name="removeEnabled">
             <bool>false</bool>
            </property>
            <property name="interactionNodeSingletonTag">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="inputA">
            <property name="text">
             <string>inputA</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="inputB">
            <property name="text">
             <string>inputB</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="qMRMLNodeComboBox" name="inputBVolumeNodeSelector">
            <property name="nodeTypes">
             <stringlist notr="true"/>
            </property>
            <property name="hideChildNodeTypes">
             <stringlist notr="true"/>
            </property>
            <property name="addEnabled">
             <bool>false</bool>
            </property>
            <property name="removeEnabled">
             <bool>false</bool>
            </property>
            <property name="interactionNodeSingletonTag">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="output">
            <property name="text">
             <string>output</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="qMRMLNodeComboBox" name="outputVolumeNodeSelector">
            <property name="nodeTypes">
             <stringlist notr="true"/>
            </property>
            <property name="hideChildNodeTypes">
             <stringlist notr="true"/>
            </property>
            <property name="baseName">
             <string/>
            </property>
            <property name="noneEnabled">
             <bool>true</bool>
            </property>
            <property name="addEnabled">
             <bool>true</bool>
            </property>
            <property name="removeEnabled">
             <bool>false</bool>
            </property>
            <property name="interactionNodeSingletonTag">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="operationComboBox"/>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="applyButton">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
